istioctl/pkg/install/verify.go

```diff
@@ -433,6 +427,15 @@ func operatorFromCluster(istioNamespaceFlag string, revision string, restClientG
 	return nil, fmt.Errorf("control plane revision %q not found", revision)
 }
 
+func convertToIstioOperator(obj map[string]interface{}) (*v1alpha1.IstioOperator, error) {
```

Comment:
I think good to move this into a [common func](https://github.com/istio/istio/blob/master/operator/pkg/apis/istio/common.go) because it will be useful at other places.

Reply:
yes `UnmarshalIstioOperator` is used in serveral other places https://github.com/istio/istio/search?q=UnmarshalIstioOperator&unscoped_q=UnmarshalIstioOperator, and I think those could all be replaced. but I am afraid that will make this PR too big to review, is it OK to replace them all in this PR?

Reply:
I could do that in a follow-up PR.

Reply:
Good to do that in a separate PR.